pkg/cephmgr/rgw/user.go

```diff
@@ -18,86 +18,146 @@ package rgw
 import (
 	"encoding/json"
 	"fmt"
-	"path"
 
-	etcd "github.com/coreos/etcd/client"
-	ctx "golang.org/x/net/context"
+	"strings"
 
-	"github.com/rook/rook/pkg/cephmgr/mon"
 	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/util"
+	"github.com/rook/rook/pkg/model"
 )
 
-const (
-	builtinUserKey = "admin"
-	idKey          = "id"
-	secretKey      = "_secret"
-)
+func ListUsers(context *clusterd.Context, runner AdminCommandRunner) ([]string, error) {
+	logger.Infof("Listing rgw users")
```

Comment:
probably not useful logging at the INFO level here either, there may be more of these